source/wp-content/themes/wporg-documentation-2022/theme.json

```diff
@@ -37,6 +46,13 @@
 					"padding": "var(--wp--preset--spacing--20)"
 				}
 			},
+			"core/post-title": {
```

Comment:
From Developer Resources https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/theme.json#L232

Reply:
Only adding this typography change but not the rest of the size updates makes the hierarchy kind of strange. Is the size change necessary for the layout?

<img width="735" alt="Screenshot 2024-06-10 at 5 55 56 PM" src="https://github.com/WordPress/wporg-documentation-2022/assets/541093/6a82cdaf-9ca0-4b21-b1c4-aa7bdda7dc56">

Reply:
No, it's necessary and I did notice that. I actually started copying all the text styles over from Dev Hub but decided it was too much for this PR. I'll revert this too and we can address it all in https://github.com/WordPress/wporg-documentation-2022/issues/87